repl/src/metta_shim.rs

```diff
@@ -84,7 +85,12 @@ impl MettaShim {
 
         //extend-py! should throw an error if we don't
         #[cfg(not(feature = "python"))]
-        new_shim.metta.tokenizer().borrow_mut().register_token_with_regex_str("extend-py!", move |_| { Atom::gnd(ImportPyErr) });
+        new_shim.metta.tokenizer().borrow_mut().register_token_with_regex_str("extend-py!", move |_| { Atom::gnd(py_mod_err::ImportPyErr) });
+
+        //Run the config.metta file
+        let repl_params = repl_params.borrow();
+        let config_metta_path = repl_params.config_metta_path();
+        new_shim.load_metta_module(config_metta_path.clone());
```

Comment:
I want this config file to contain all the settings for both interactive and script-execution mode.  However, there are currently only interactive settings because I don't see a way to do the string manipulation for import paths using MeTTa's current stdlib.

Is you point that running these setup statements is a waste of time if they don't matter for the execution mode?  We could do something to separate them.

What do you think?  A separate repl.metta file (runs only in interactive mode), in addition to the config.metta file (runs always)?

Reply:
Well, I think it intersects heavily with pragmas which was implemented by @Necr0x0Der , thus we need to think what we should keep or how we can merge these two ways of configuration. I don't like we have two different mechanisms to tune the execution. Keeping settings inside an atomspace looks more logical to me. But then we need to align pragma syntax to change these settings inside this atomspace.

Reply:
I will wait for your discussion with Alexey.

Reply:
I have raised https://github.com/trueagi-io/hyperon-experimental/issues/425 to address this in future